src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -25,6 +28,10 @@ private void Render(List<CodeLine> list, IEnumerable<CodeFileToken> node, bool s
             bool isDocumentationRange = false;
             bool isDeprecatedToken = false;
             bool isSkipDiffRange = false;
+            Stack<(string, string)> nodesInProcess = new Stack<(string, string)>();
```

Comment:
To improve the readability of the code, it will be good to rethink this type. It is not clear what is in `Item1` and what in `Item2`.
Consider creating a struct for that type

Reply:
Updated with new logic.